recipes/festival/meta.yaml

```
@@ -0,0 +1,97 @@
+
+build:
+  run_exports:
+    - {{ pin_subpackage('festival', max_pin='x.x') }}
+  skip: True  # [not linux]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make  # [linux]
+    - libgomp  # [linux]
+    - libestools-dev  # [linux]
+    - libestools-static  # [linux]  This is needed to satisfy Make, but not used
+
+  host:
+    - libestools  # [linux]
+  run:
+    - libestools  # [linux]
+
+outputs:
+  - name: festival
+    script: install.sh  # [linux]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - libgomp  # [linux]
+        - libestools-dev  # [linux]
+      host:
+        - libestools  # [linux]
+      run:
+        - libestools  # [linux]
+    test:
+      commands:
+        - test -f $PREFIX/bin/festival  # [linux]
+        - test -f $PREFIX/bin/festival_client  # [linux]
+        - test -f $PREFIX/bin/text2wave  # [linux]
+        - test -f $PREFIX/lib/init.scm  # [linux]
+        - festival --version  # [linux]
+        - festival_client --version  # [linux]
+        - text2wave -h  # [linux]
+
+  - name: festival-dev
+    script: install_devel.sh  # [linux]
+    requirements:
+      host:
+        - {{ pin_subpackage("festival", exact=True) }}
+      run:
+        - {{ pin_subpackage("festival", exact=True) }}
+    test:
+      commands:
+        - test -f $PREFIX/include/festival/festival.h  # [linux]
+
+  - name: festival-static
+    script: install_static.sh  # [linux]
+    requirements:
+      host:
+        - {{ pin_subpackage("festival", exact=True) }}
+      run:
+        - {{ pin_subpackage("festival", exact=True) }}
+    test:
+      commands:
+        - test -f $PREFIX/lib/libFestival.a  # [linux]
+
+about:
+  home: https://github.com/festvox/festival
+  summary: 'Festival offers a general framework for building speech synthesis systems as well as including examples of various modules'
+  description: |
+    Festival offers a general framework for building speech synthesis systems as well as including examples of various
+    modules. As a whole it offers full text to speech through a number APIs: from shell level, though a Scheme command
+    interpreter, as a C++ library, and an Emacs interface. Festival is multi-lingual (currently English (US and UK)
+    and Spanish are distributed but a host of other voices have been developed by others) though English is the most
+    advanced.
+    The system is written in C++ and uses the Edinburgh Speech Tools for low level architecture and has a Scheme (SIOD)
+    based command interpreter for control. Documentation is given in the FSF texinfo format which can generate, a
+    printed manual, info files and HTML.
+  license: BSD-4-Clause
+  license_family: BSD
+  license_file: COPYING
+  doc_url: https://www.cstr.ed.ac.uk/projects/festival
+  dev_url: https://github.com/festvox/festival
+
+extra:
+  recipe-maintainers:
+    - peci1
```

```suggestion
extra:
  feedstock-name: festival
  recipe-maintainers:
    - peci1
```
Otherwise, the feedstock will be named festival-split